docs/retrieve.txt

```diff
@@ -0,0 +1,360 @@
+.. _laravel-fundamentals-retrieve:
+
+==============
+Retrieve Data
+==============
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: find one, find many, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the Laravel package to perform **find operations**
+on your MongoDB collections. Find operations allow you to retrieve documents based on
+criteria that you specify.
+
+This guide shows you how to perform the following tasks:
+
+- :ref:`laravel-retrieve-matching`
+- :ref:`laravel-retrieve-all`
+- :ref:`Modify Find Operation Behavior <laravel-modify-find>`
+
+Prerequisites
+-------------
+
+Complete the :ref:`Quick Start <laravel-quick-start>` tutorial before running the examples in this
+guide.
+
+After completing the Quick Start, ensure that your MongoDB deployment and Laravel application
+satisfy the following requirements:
+
+- Your Atlas MongoDB deployment contains the Atlas sample data
+- ``Movie.php`` file contains a ``Movie`` model to represent the ``movies`` collection
+- ``MovieController.php`` file contains a ``show()`` function to run database operations
+- ``browse_movies.blade.php`` file contains HTML code to display the results of database
+  operations
+
+The following sections describe how to edit the files in your Laravel application to run 
+the code examples and view the expected output.
+
+.. _laravel-retrieve-matching:
+
+Retrieve Documents that Match a Query
+-------------------------------------
+
+You can retrieve documents that match a set of criteria by passing a query filter to the ``where()``
+method. A query filter specifies field value requirements and instructs the find operation
+to only return documents that meet these requirements. To run the query, call the ``where()``
+method on an Eloquent model or query builder that represents your collection.
+
+You can pass the following types of query filters to the ``where()`` method:
+
+- ``where('<field name>', <value>)``: instructs MongoDB to return documents that match
```

Comment:
I find it more explicit if the context of the function is provided. Otherwise they can think the function `where` is global.

```suggestion
- ``Model::where('<field name>', <value>)``: instructs MongoDB to return documents that match
```